app/js/lib/foxbox/foxbox.js

```diff
@@ -355,7 +355,7 @@ export default class Foxbox extends Service {
     return this[p.net].fetchJSON(`${this[p.net].origin}/services/${id}/state`)
       .then(res => {
         if (!res) {
-          throw new Error(`The action couldn't be performed.`);
+          throw new Error('The action couldn\'t be performed.');
```

Comment:
I believe if we don't use parameters inside ```template string``` it complains about it - that's good IMO, cause it's just string no need in anything else. If we have parameters then using ```template string``` is good.

Does it make sense?